app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,27 @@
+class RentsController < ApplicationController
+
+  #before_action :authenticate_user!
+
+
+  def create
+    rent = Rent.new(create_params.merge(user: current_user) )# with this we can be sure the rent will be associated to the current_user
+    rent.save
+
+    if rent.save
+      mail = RentMailer.rent_email(rent)
+      mail.deliver_later
+      render json: rent
+    else
+      puts "Sad path"
```

Comment:
What if we add a real sad path 😛 Printing in the screen "sad path" will not give any data to the consumer of our API. Take a look at this http://travisjeffery.com/b/2012/04/rendering-errors-in-json-with-rails/